lib/sorted/active_record/builder.rb

```diff
@@ -72,7 +72,14 @@ def to_hash
       end
 
       def to_sql
-        @return_hash ? to_hash : ::Sorted::SQLQuery.encode(@set)
+        # Use string in case one of values is nulls first/last
+        @return_hash && no_nulls_ordering? ? to_hash : ::Sorted::SQLQuery.encode(@set)
```

Comment:
what is the risk of SQL injection if no nulls order exists?

Reply:
Checking `Sorted` classes that are being used, it's not possible to parse a different query from this format: table_name.column + direction + nulls first or last. And since the app layer is whitelisting sorting fields, I don't believe an injection would be possible.